helm/harbor-scanner-trivy/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: harbor-scanner-trivy
 version: 0.1.1
```

Comment:
Don't you have to update the chart version?

Reply:
I think it's fine just to bump up appVersion. We do not change K8S deployment descriptors.

Reply:
I think `helm upgrade` is ignored unless the version is not bumped up, but it may be my misunderstanding.

Reply:
Let me test / double check `helm upgrade` to confirm ...

Reply:
It looks fine if it works without bumping up the version.